code/modules/mining/abandoned_crates.dm

```diff
@@ -20,135 +20,104 @@
-		if(67 to 68)
-			for(var/i in 1 to rand(4, 7))
-				var/newitem = pick(subtypesof(/obj/item/stock_parts) - /obj/item/stock_parts/subspace)
-				new newitem(src)
-		if(69 to 70)
-			new /obj/item/stack/ore/bluespace_crystal(src, 5)
-		if(71 to 72)
-			new /obj/item/pickaxe/drill(src)
-		if(73 to 74)
-			new /obj/item/pickaxe/drill/jackhammer(src)
-		if(75 to 76)
-			new /obj/item/pickaxe/diamond(src)
-		if(77 to 78)
-			new /obj/item/pickaxe/drill/diamonddrill(src)
-		if(79 to 80)
-			new /obj/item/cane(src)
-			new /obj/item/clothing/head/collectable/tophat(src)
-		if(81 to 82)
-			new /obj/item/gun/energy/plasmacutter(src)
-		if(83 to 84)
-			new /obj/item/toy/katana(src)
-		if(85 to 86)
-			new /obj/item/defibrillator/compact(src)
-		if(87) //1% chance
-			new /obj/item/weed_extract(src)
-		if(88)
-			new /obj/item/organ/brain(src)
-		if(89)
-			new /obj/item/organ/brain/alien(src)
-		if(90)
-			new /obj/item/organ/heart(src)
-		if(91)
-			new /obj/item/device/soulstone/anybody(src)
-		if(92)
-			new /obj/item/katana(src)
-		if(93)
-			new /obj/item/dnainjector/xraymut(src)
-		if(94)
-			new /obj/item/storage/backpack/clown(src)
-			new /obj/item/clothing/under/rank/clown(src)
-			new /obj/item/clothing/shoes/clown_shoes(src)
-			new /obj/item/device/pda/clown(src)
-			new /obj/item/clothing/mask/gas/clown_hat(src)
-			new /obj/item/bikehorn(src)
-			new /obj/item/toy/crayon/rainbow(src)
-			new /obj/item/reagent_containers/spray/waterflower(src)
-		if(95)
-			new /obj/item/clothing/under/rank/mime(src)
-			new /obj/item/clothing/shoes/sneakers/black(src)
-			new /obj/item/device/pda/mime(src)
-			new /obj/item/clothing/gloves/color/white(src)
-			new /obj/item/clothing/mask/gas/mime(src)
-			new /obj/item/clothing/head/beret(src)
-			new /obj/item/clothing/suit/suspenders(src)
-			new /obj/item/toy/crayon/mime(src)
-			new /obj/item/reagent_containers/food/drinks/bottle/bottleofnothing(src)
-		if(96)
-			new /obj/item/hand_tele(src)
-		if(97)
-			new /obj/item/clothing/mask/balaclava
-			new /obj/item/gun/ballistic/automatic/pistol(src)
-			new /obj/item/ammo_box/magazine/m10mm(src)
-		if(98)
-			new /obj/item/katana/cursed(src)
-		if(99)
-			new /obj/item/storage/belt/champion(src)
-			new /obj/item/clothing/mask/luchador(src)
-		if(100)
-			new /obj/item/clothing/head/bearpelt(src)
+/*
+HOW TO DO PROBABILITY WITH PICK: pick(P;Val) where P is the weight (think of is as if there are "P" identical entries, where P is a number
+Note: default internal P is 100. So if you don't bother adding a P to every argument then to get something twice as much you need to use 200.
+1: you can also embed pick() in itself (with or with out numbers) to create a secondary pool of items as a result
+2: use list() for multiple items bundled together as a possible result
+3: alternatively use list() with an associated value to generate multiple copies. example: list(/obj/=10) = 10 copies.of /obj
+*/
+
+	var/loot = pick(
+		//5%
```

Comment:
Just to be clear, these comments are only true because the total relative values add up to be 100

Any additional items added will make all related comments incorrect.